UnityProject/Assets/Scripts/ScriptableObjects/RP/EmoteSO.cs

```diff
@@ -60,6 +82,24 @@ public virtual void Do(GameObject player)
 			}
 		}
 
+		/// <summary>
+		/// Use this instead of rewriting Chat.AddActionMsgToChat() when adding text to a failed conditon.
+		/// </summary>
+		/// <param name="player">The orignator</param>
+		/// <param name="type">0: failText, 1: critViewText</param>
+		protected void FailText(GameObject player, int type)
```

Comment:
that int type is a little obscure. You can do a private enum to make it more human readable.

Reply:
Done 👍